pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -186,6 +193,24 @@ func (ctrl *Controller) syncNodeConfigHandler(key string) error {
 	return nil
 }
 
+func (ctrl *Controller) syncNodeConfigStatus(cfg *osev1.Node) error {
+	statusUpdateErr := retry.RetryOnConflict(updateBackoff, func() error {
+		newStatusCondition := apihelpers.NewCondition(cgroupModeCondType, metav1.ConditionTrue, cgroupCondReason, cgroupsV1DeprecationMsg)
+		if len(cfg.Status.Conditions) == 0 || newStatusCondition.Message != cfg.Status.Conditions[len(cfg.Status.Conditions)-1].Message {
+			cfg.Status.Conditions = append(cfg.Status.Conditions, *newStatusCondition)
+		} else if cfg.Status.Conditions[len(cfg.Status.Conditions)-1].Message == newStatusCondition.Message {
+			cfg.Status.Conditions[len(cfg.Status.Conditions)-1] = *newStatusCondition
+		}
+		_, updateErr := ctrl.configClient.ConfigV1().Nodes().UpdateStatus(context.TODO(), cfg, metav1.UpdateOptions{})
```

Comment:
Question: Will this cause repeated update status API calls? I maybe missing it but I don't see a check to compare against the existing condition.

Reply:
No, this shouldn't be repetitive call. This gets updated with the updated status object whenever there is a sync in the node config object

Reply:
ack, sounds good to me (: